rusk-abi/src/abi.rs

```diff
@@ -70,3 +72,9 @@ pub fn payment_info(
 ) -> Result<crate::PaymentInfo, ContractError> {
     call(contract, "payment_info", &())
 }
+
+/// Query owner of a given contract.
+#[cfg(feature = "abi")]
+pub fn owner_raw(contract: ContractId) -> Option<OwnerRaw> {
+    owner(contract)
+}
```

Comment:
There's no reason to come up with the type alias. Just use the length directly. It even becomes more clear tbh

```suggestion
/// Query owner of a given contract.
#[cfg(feature = "abi")]
pub fn owner_raw(contract: ContractId) -> Option<[u8; PublicSpendKey::SIZE]> {
    owner(contract)
}
```

Reply:
I do have to say that since you're still directly exposing `piecrust_uplink::owner` and `self_owner` directly, this name hardly makes much sense.